start-tests.sh

```
@@ -151,9 +113,8 @@ if [[ ! -z $CYPRESS_OPTIONS_HUB_PASSWORD && "$CYPRESS_OPTIONS_HUB_PASSWORD" != "
     server="${server}:6443"
   fi
 
-  oc login --server=${server} -u $CYPRESS_OPTIONS_HUB_USER -p $CYPRESS_OPTIONS_HUB_PASSWORD --insecure-skip-tls-verify
+  oc login --server=${server} -u $OPTIONS_HUB_USER -p $OPTIONS_HUB_PASSWORD --insecure-skip-tls-verify
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)